opm/material/fluidmatrixinteractions/EclEpsConfig.hpp

```diff
@@ -196,8 +196,9 @@ class EclEpsConfig
                       const Opm::EclipseState& eclState,
                       Opm::EclTwoPhaseSystemType twoPhaseSystemType)
     {
+        const auto& endscale = eclState.runspec().endpoint_scaling();
```

Comment:
isn't the `endpoint_scaling()` method name *quite* inconsistent with the rest of the parser?